components/suggest/src/yelp.rs

```diff
@@ -270,36 +301,36 @@ impl SuggestDao<'_> {
     /// (
     ///   String: The keyword in DB (but the case is inherited by query).
     ///   bool: Whether or not the keyword is exact match.
+    ///   SubjectType: The subject type.
     ///   &[&str]: Words after removed matching subject.
     /// )
```

Comment:
And then please update this comment for the new struct type. I would probably move these comments up to the struct definition itself.